elements/lisk-validator/test/validation.spec.ts

```diff
@@ -79,7 +79,7 @@ describe('validation', () => {
 	describe('#isHexString', () => {
 		it('should return true when valid hex was provided', () => {
 			return expect(
-				isHexString('215b667a32a5cd51a94c9c2046c11fffb08c65748febec099451e3b164452bc'),
+				isHexString('215b667a32a5cd51a94c9c2046c11fffb08c65748febec099451e3b164452b'),
```

Comment:
Now that we're mentioning hex strings in UPPERCASE, it wouldn't be such a bad idea to update this test case to provide input string in MiXeDCaSe, or create a new one which has MiXeDCaSe or UPPERCASE.